src/layouts/Sidebar.tsx

```diff
@@ -36,13 +36,16 @@ function Sidebar(this: any) {
             Create
           </Button>
           <Button color="inherit">Log out</Button>
+          <Button component={Link} to={"/auth"} color="inherit">
+            Log in
+          </Button>
```

Comment:
`Log in` button should be moved outside of Sidebar's Drawer content. It should replace `Open` button for not logged in users. Use a prop like `userData` in this component and show different buttons based on user data presence. For now lets manually change it to see the difference.